packages/enrichPrescriptions/tests/utils.ts

```diff
@@ -126,3 +127,23 @@ export function defaultExtension(onboarded: boolean = true): Array<Extension> {
     ]
   }]
 }
+
+export function addExtensionToMedicationRequest(bundle: Bundle, status: string, statusDate: string) {
```

Comment:
This function is more addExtensionToTheFirstMedicationRequestInTheFirstBundleInABundle. I'd suggest that it should accept a MedicationRequest.